main.c

```diff
@@ -48,11 +48,14 @@ void print_help(char* app_name) {
 
 int main(int argc, char* argv[]) {
   char *librem_exec = "libremkey";
+  char *nitrokey_exec = "nitrokey";
   char *key_brand;
   if(strstr(argv[0], librem_exec) != NULL) {
     key_brand = "Librem Key";
-  } else {
+  } else if(strstr(argv[0], nitrokey_exec) != NULL) {
     key_brand = "Nitrokey";
+  } else {
+    key_brand = "USB security dongle";
```

Comment:
Just an idea by the way: brand can be taken from the used device, and stored for the later use for UI needs. No need to have it hard-coded in multiple binaries, unless there is a need to accept only one particular device model.

Reply:
I like the idea of detecting (when possible) the actual device in use and branding it in the interface accordingly. My main worry with generic branding is the risk of confusing a user who is new to these technologies and doesn't see "USB security dongle" printed on their Nitrokey or Librem Key, but does see "Nitrokey" or "Librem Key" on there. Since the USA Librem Keys and Nitrokeys now have different USB IDs we should be able to tell them apart relatively easily.